### PR TITLE
Rename migInstancesState to migInstancesStateCount in gce cache

### PR DESCRIPTION
**Pull request by damikag**
#### What type of PR is this?
/kind cleanup

#### What this PR does / why we need it:
This PR renames ```migInstancesState``` to ```migInstancesStateCount``` in gce cache
#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?
NONE

```release-note

```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

File: cluster-autoscaler/cloudprovider/gce/cache.go [hunk 1]
```diff
@@ -549,24 +549,24 @@ func (gc *GceCache) InvalidateAllListManagedInstancesResults() {
 	gc.listManagedInstancesResultsCache = make(map[GceRef]string)
 }
 
-// GetMigInstancesState returns instancesState for the given mig from cache.
-func (gc *GceCache) GetMigInstancesState(migRef GceRef) (instanceState map[cloudprovider.InstanceState]int64, found bool) {
+// GetMigInstancesStateCount returns instancesState for the given mig from cache.
```
Comment (bskiba) [inline cluster-autoscaler/cloudprovider/gce/cache.go:552]:
```suggestion
// GetMigInstancesStateCount returns counts of instances in different states for the given mig from cache.
```

**Review by bskiba — COMMENTED** [review]
/lgtm
/approve

File: cluster-autoscaler/cloudprovider/gce/cache.go (hunk 1 above)
Comment (damikag) [reply cluster-autoscaler/cloudprovider/gce/cache.go:552]: Done

**Pull request merged**